php/add-subscriber-to-list.php

```diff
@@ -0,0 +1,70 @@
+<?php
+  /*
+   * Add a 'member' to a 'list' via mailchimp API v3.x
+   *
+   * @ http://developer.mailchimp.com/documentation/mailchimp/reference/lists/members/#create-post_lists_list_id_members
+   *
+   * Background:
+   * Typical use case is that this code would get run by an .ajax() jQuery call or possibly a form action
+   * The live data you need will get transferred via the global $_POST variable
+   * That data must be put into an array with keys that match the mailchimp endpoints, check the above link for those
+   * You also need to include your API key and list ID for this to work.
+   * You'll just have to go get those and type them in here, see README.md
+   *
+   */
+
+  // Set API Key and list ID to add a subscriber
+  $api_key = 'your-api-key-here';
+  $list_id = 'your-list-id-here';
+
+  // Setup destination URL
+  // Note that your API URL has that location subdomain up front
+  // It can vary, check the last 3 digits of your API key
+  $url = 'https://us5.api.mailchimp.com/3.0/lists/' . $list_id . '/members/';
+
+  // Encode data into a format that the add subscriber mailchimp end point is looking for
+  // Must include 'email_address' and 'status'
+  // Statuses: pending = they get an email; subscribed = they don't get an email
+  // Any custom fields go into the weirdly named 'merge_fields'
```

Comment:
It's not so "weirdly" as these are the field names used for merge functionality within templates used by both the Mailchimp admin UI and Mandrill.

Search for "merge_vars":
- https://mandrillapp.com/api/docs/messages.JSON.html#method=send-template 

I encourage you to remove your opinion comments from the PR. Fun to read but perhaps misleading.